projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -94,14 +95,7 @@ export default function SubscriptionControls( {
 						},
 						{
 							colorValue: borderColor.color,
-							onColorChange: newBorderColor => {
-								// Note: setBorderColor from withColors hook does not
-								// work correctly with shortcode border color rendering.
-								setAttributes( {
-									borderColor: newBorderColor,
-									customBorderColor: newBorderColor,
-								} );
```

Comment:
Inspecting the current HTML results of this PR I don't see any issue with using `setBorderColor` although I do not know what exactly was the issue before with the comment written above.

And just pulling a possibly related comment in here. https://github.com/Automattic/jetpack/pull/18993#issuecomment-793265578

Reply:
Yeah, I haven't noticed any issues either.